### DISCUSSION TITLE
跳转之后没有样式了怎么解决呀

### DISCUSSION THREAD

**Discussion by Pandada1998** [liuyib/hexo-theme-stun#212] [Q&A（反馈问题、提问、报告 Bug 等）]
第一次用，主页有样式，但是跳转到其他页面，比如归档，分类等就没有样式了，控制台报错如下
![image](https://user-images.githubusercontent.com/56426095/125165780-144af700-e1cb-11eb-8ab6-a23650d5ef23.png)

**Comment by liuyib** [comment]
这个报错无法定位错误，所有资源都 404，很可能是你的 Hexo 配置设置有误。我猜的是你的 Hexo 配置中，没有将网站放在根目录下。将博客运行在非根目录下，我记得 Stun 目前是不支持的。@Pandada1998
